f2/src/forms/ContactInfoForm.js

```diff
@@ -25,6 +25,9 @@ import { Link as ReactRouterLink } from 'react-router-dom'
 export const validate = (values) => {
   const errors = {}
 
+  //a loose validation for phone number(only allow numbers, space, brackets and hyphen)
+  const phoneRegex = /^[0-9()\-\s]*$/g
```

Comment:
This does allow more flexibility, but I can enter one digit. The form says to enter a valid phone number, in my opinion 1 is just as invalid as T,.

Reply:
This is a loose validation to meet Hong's requirement "at least validated to accept the digit numbers only and remove phrase (Enter a phone number that contains 10 digits.)"